src/titiler/extensions/tests/test_wms.py

```diff
@@ -0,0 +1,373 @@
+
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.3.0",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "373185.0,8019284.949381611,639014.9492102272,8286015.0",
+                "SRS": "EPSG:32621",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/tiff; application=geotiff",
+            },
+        )
+        assert response.status_code == 200
+        meta = parse_img(response.content)
+        assert meta["crs"] == CRS.from_epsg(32621)
+
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.3.0",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "373185.0,8019284.949381611,639014.9492102272,8286015.0",
+                "CRS": "EPSG:32621",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/tiff; application=geotiff",
+            },
+        )
+        assert response.status_code == 200
+        meta = parse_img(response.content)
+        assert meta["driver"] == "GTiff"
+        assert meta["width"] == 334
+        assert meta["height"] == 333
+        assert meta["crs"] == CRS.from_epsg(32621)
+
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.3.1",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "373185.0,8019284.949381611,639014.9492102272,8286015.0",
+                "CRS": "EPSG:32621",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/png",
+            },
+        )
+        assert response.status_code == 400
+        assert "Invalid 'VERSION' parameter" in response.text
+
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.3.0",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "373185.0,8019284.949381611,639014.9492102272,8286015.0",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/png",
+            },
+        )
+        assert response.status_code == 400
+        assert "Missing 'CRS' or 'SRS parameters." in response.text
+
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.3.0",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "373185.0,373185.0,8019284.949381611,639014.9492102272,8286015.0",
+                "CRS": "EPSG:32621",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/png",
+            },
+        )
+        assert response.status_code == 400
+        assert "Invalid 'BBOX' parameters" in response.text
+
+        # 1.3.0 needs inverted X,Y coordinates for EPSG:4326
```

Comment:
Awesome checking for the messed up coordinate swapping when switching to 1.3.0 and 4326